tests/unit-tests/admin/test-class-sensei-onboarding.php

```diff
@@ -40,6 +40,169 @@ public function testClassInstance() {
 		$this->assertTrue( class_exists( 'Sensei_Onboarding' ), 'Sensei Onboarding class does not exist' );
 	}
 
+	/**
+	 * Test onboarding wizard notice in dashboard.
+	 *
+	 * @covers Sensei_Onboarding::onboarding_wizard_notice
+	 * @covers Sensei_Onboarding::is_current_screen_selected_to_wizard_notice
+	 */
+	public function testOnboardingWizardNoticeInDashboard() {
+		// Create and login as admin.
+		$admin_id = $this->factory->user->create( array( 'role' => 'administrator' ) );
+		wp_set_current_user( $admin_id );
+
+		set_current_screen( 'dashboard' );
+		update_option( \Sensei_Onboarding::SUGGEST_ONBOARDING_OPTION, 1 );
+
+		ob_start();
+		Sensei()->onboarding->onboarding_wizard_notice();
+		$html = ob_get_clean();
+
+		$this->assertNotEmpty( $html, 'Should return the notice HTML' );
```

Comment:
Should we look for some text in the HTML just to make sure it isn't responding with something non-empty but wrong? For example, maybe look for the correct button text?

(This applies to several tests below as well).

Reply:
Makes sense! https://github.com/Automattic/sensei/pull/3096/commits/0a0a552b147c54cc9f9c8f73a09051182764a5d9